openlibrary/tests/core/test_sponsors.py

```diff
@@ -44,22 +44,22 @@ def test_do_we_want_it(self, monkeypatch, mock_site):
         # Simulating exception / API call failure ...
         monkeypatch.setattr(sponsorships.lending, 'get_work_availability',
                             lambda work_id: mock_availability__need)
-        monkeypatch.setattr(sponsorships.requests, 'get', lambda url: {'invalid': 'json'})
```

Comment:
Why did these change?

Reply:
We switched to a different form of the function, so the mock has to change to match.

Reply:
Using kwargs might be a better option here, so we don't have to be updating it all the time:

```py
monkeypatch.setattr(sponsorships.requests, 'get', lambda url, **kwargs: {'invalid': 'json'}) 
```

Reply:
Sure, whatever. Changed in 804739c33ef320d87f9233c107b9f57087e90e69